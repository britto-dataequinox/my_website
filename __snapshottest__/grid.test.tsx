import React from 'react';
import renderer from 'react-test-renderer';
import ResponsiveGrid from '../components/Grid';

test('grid snap', () => {
  const tree = renderer.create(
    <ResponsiveGrid>
      <div>Item 1</div>
      <div>Item 2</div>
    </ResponsiveGrid>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
