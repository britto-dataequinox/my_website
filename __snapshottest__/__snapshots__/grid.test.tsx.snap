// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grid snap 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
>
  <div>
    Item 1
  </div>
  <div>
    Item 2
  </div>
</div>
`;
